### PR TITLE
About page revamp

### PR DESCRIPTION
**Pull request by PikaDude**
Revamps the about page under options to include:
- A dynamic version number from the extension's manifest
- A dynamic contributor table from GitHub (featuring usernames, avatars and links to profiles)
- Updates to the buttons
- Moves some CSS classes manually added to chrome-bootstrap to a seperate file
- Fixes the race condition in options.js, now it waits for the page to load before attempting to do anything

### CODE REVIEW

File: css/chrome-bootstrap.css [hunk 1]
```diff
@@ -683,32 +683,4 @@
 .chrome-bootstrap .overlay .page .button-strip button {
   -webkit-margin-start: 10px;
   display: block;
-}
-
-/* Added by Andrex. */
```
Comment (PikaDude) [inline css/chrome-bootstrap.css:688]: Moved to a seperate file.

File: manifest.json [hunk 2]
```diff
@@ -39,5 +39,5 @@
       "http://api.openweathermap.org/"
    ],
    "update_url": "https://clients2.google.com/service/update2/crx",
-   "version": "3.4"
+   "version": "4.0"
```
Comment (PikaDude) [inline manifest.json:42]: Incremented version number to 4.0 as discussed.

File: options.html [hunk 3]
```diff
@@ -173,43 +180,36 @@ <h1>About</h1>
...
+							<br />
+							<span id="version-number"></span>
 						</div>
 					</div>
 
 					<div>
-						<button id="get-help" role="link" href="https://chrome.google.com/webstore/detail/animal-crossing-music/fcedlaimpcfgpnfdgjbmmfibkklpioop/support">Get help with using this extension</button>
-						<button id="report-an-issue" role="link" href="https://chrome.google.com/webstore/detail/animal-crossing-music/fcedlaimpcfgpnfdgjbmmfibkklpioop/support">Report an issue</button>
-						<button id="help-us-out" role="link" href="https://github.com/JdotCarver/Animal-Crossing-Music-Extension/">Help us make the best AC extension ever!</button>
+						<a target="_blank" href="https://discord.gg/4FMrEF8"><button role="link">Join our Discord Server to get help using this extension</button></a>
```
Comment (PikaDude) [inline options.html:193]: Updates the button names and links to be more relevant and up-to-date.

File: manifest.json (hunk 2 above)
Comment (jordigarcl) [reply manifest.json:42]: 👍

File: scripts/options/options.js [hunk 4]
```diff
@@ -88,37 +123,7 @@ function restoreOptions() {
...
-	'kk-version-both'
-];
-
-document.getElementById('volume').oninput = saveOptions;
-
-onClickElements.forEach(el => {
-	document.getElementById(el).onclick = saveOptions;
-});
-
-let updateLocationEl = document.getElementById('update-location');
```
Comment (jordigarcl) [inline scripts/options/options.js:120]: Was this line removed by mistake?  I'm getting the following error when clicking the UpdateLocation button:
```
Uncaught ReferenceError: updateLocationEl is not defined
    at HTMLButtonElement.validateWeather (options.js:127)
```

**Review by jordigarcl — CHANGES_REQUESTED** [review]
I think you inadvertently broke Location Update. However, great work, I love the dynamic contribution table!

**Review by jordigarcl — APPROVED** [review]

**Pull request merged**